tokio/Cargo.toml

```diff
@@ -146,7 +146,7 @@ mockall = "0.11.1"
 async-stream = "0.3"
 
 [target.'cfg(not(any(target_arch = "wasm32", target_arch = "wasm64")))'.dev-dependencies]
-socket2 = "0.4"
+socket2 = "0.4.9"
```

Comment:
It occurred to me that this bump probably wasn't necessary, but it doesn't really change anything.